### DISCUSSION TITLE
Questions about workflow: export and subfolders

### DISCUSSION THREAD

**Discussion by anthrolisp** [tiddly-gittly/TidGi-Desktop#233] [Q&A] [answered]
How to create an html edition of a specific selection of tiddlers?

Is it possible to keep different tiddlers in their own separate folder (inside the project folder)?

**Comment by linonetwo** [comment, marked as answer]
> create an html edition

For tw related knowledge you can just google https://groups.google.com/g/tiddlywiki/c/FLUr-QI7Qv0

There are decade of posts there.

> keep different tiddlers in their own separate folder

You mean creating `tiddlers/type1/AAA` and `tiddlers/type2/BBB` ?

Just make your tiddler with `type1/AAA` then there will be a folder `tiddlers/type1` created.

You can also ask question in talk.tiddlywiki.org/ ! I'm always there too.

**Reply by anthrolisp** [reply]
Thank you for the reply! Re folders, is there a way to do this through the TiddlyWiki or TidGi interface?

**Reply by linonetwo** [reply]
You mean open the folder? just right click the icon.
<img width="268" alt="截屏2022-05-10 23 20 58" src="https://user-images.githubusercontent.com/3746270/167664095-a4a05cb2-2395-46b4-b82c-353b45588660.png">
 
If mean create the folder, that is a tw feature, you can ask in the talk forum!

**Reply by anthrolisp** [reply]
Thank you! Is it possible to set which folder a tiddler lives when creating/editing it through TiddlyWiki user interface? Thank you again!

**Reply by linonetwo** [reply]
I think that is the filesystempath , see official doc maybe.
